google_shopping_functions.js

```diff
@@ -1,30 +1,82 @@
 //define getItemsCount function
 //accepts full item data
 //returns the length of the items array
+var data = require("./products.json")
+
 var getItemsCount = function(itemData){
+	for(items in itemData){
```

Comment:
Here, we are just looking for a count (the length of the entire array), so we aren't actually looping through anything. You can take out the for loop and just keep the return statement here.